propeoplemd/cibox
# Cleanup Apache2 custom code

This PR provides functionality to go away from custom code for apache2.

How to test:
- Provision new server
- Make sure Jenkins is available at http://SERVER_IP:8080
- Make sure `/var/www` at http://SERVER_IP (just create some files there in order to test, otherwise you'll see Access Denied)
- Make sure `https` works for https://SERVER_IP/*
